healthcheck-sofa-boot-starter/src/test/java/com/alipay/sofa/healthcheck/bean/ReferenceB.java

```diff
@@ -26,9 +26,17 @@
  */
 public class ReferenceB implements HealthChecker {
 
-    private static boolean isStrict;
+    private boolean isStrict;
 
-    private static int     retryCount;
+    private int     retryCount;
+
+    public ReferenceB() {
```

Comment:
Is this default constructor necessary?

Reply:
I see, but there are not used currently? May be we should add default constructor and setters when needed.